### PR TITLE
adds VADR 1.5

### PR DESCRIPTION
**Pull request by kapsakcj**
This PR adds a dockerfile and README.md for VADR v1.5. Docker image builds successfully. Will address #462 

Most changes from v1.4.2 are MPXV related except for a few bug fixes. https://github.com/ncbi/vadr/blob/master/RELEASE-NOTES.md

The dockerfile is slightly changed from the v1.4.2 dockerfile. Changes:

- line 20 is new, due to new minimap2 dependency ` VADRMINIMAP2DIR=$VADRINSTALLDIR/minimap2`
- added `apt-get install --no-install-recommends` in an attempt to reduce the image size. Saved a small amount of disk space. Image is about the same size as 1.4.2 image despite adding `minimap2`
- added `apt-get install ca-certificates procps zlib1g-dev` 
  - `procps` to ensure `ps` is installed for nextflow
  - Zlib dev files are required for compiling minimap2
  - `ca-certificates` to ensure `wget` runs without issue
- added new MPXV recommended options `v-annotate.pl --minimap2 --s_overhang 150` for MPXV test at end of dockerfile. Also added this to the example in the README.md

SARS-CoV-2 recommended options have not changed. Also the VADR models for SC2 and MPXV have not changed.

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [x] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [x] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: vadr/1.5/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+ VADRMINISCRIPTSDIR=$VADRINSTALLDIR/vadr/miniscripts \
+ VADRMODELDIR=$VADRINSTALLDIR/vadr-models \
+ VADRINFERNALDIR=$VADRINSTALLDIR/infernal/binaries \
+ VADREASELDIR=$VADRINSTALLDIR/infernal/binaries \
+ VADRHMMERDIR=$VADRINSTALLDIR/hmmer/binaries \
+ VADRBIOEASELDIR=$VADRINSTALLDIR/Bio-Easel \
+ VADRSEQUIPDIR=$VADRINSTALLDIR/sequip \
+ VADRBLASTDIR=$VADRINSTALLDIR/ncbi-blast/bin \
+ VADRFASTADIR=$VADRINSTALLDIR/fasta/bin \
+ VADRMINIMAP2DIR=$VADRINSTALLDIR/minimap2
```
Comment (erinyoung) [inline vadr/1.5/Dockerfile:20]: So many environmental variables 😵

File: vadr/1.5/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+ /opt/vadr/vadr/miniscripts/fasta-trim-terminal-ambigs.pl \
+        /test-data/SRR13957123.consensus.fa \
+        --minlen 50 \
+        --maxlen 30000 \
+        > /test-data/SRR13957123.consensus.trimmed.fasta && \
+ v-annotate.pl --noseqnamemax --glsearch -s -r --nomisc \
+       --mkey sarscov2 --lowsim5seq 6 --lowsim3seq 6 --alt_fail lowscore,insertnn,deletinn \
+       "/test-data/SRR13957123.consensus.trimmed.fasta" \
+       "SRR13957123-vadr-outdir" && \
+  ls SRR13957123-vadr-outdir
```
Comment (erinyoung) [inline vadr/1.5/Dockerfile:111]: The tab justification here looks a little off for some reason. It's not something that I'll make you change, just wanted to let you know.

**Comment by erinyoung** [comment]
The files look okay (there are some tab issues that make your file less pretty, but I'm not going to make you change those), and it looks like the tests worked well.

I'm going to approve and merge this file. 
I will then push this to dockerhub and quay and overwrite the 'latest' tag.

**Review by erinyoung — APPROVED** [review]

File: vadr/1.5/Dockerfile (hunk 2 above)
Comment (kapsakcj) [reply vadr/1.5/Dockerfile:111]: whoops! I can make the lines aligned/cleaner in the next dockerfile version. I was mainly just trying to break up the main commands for readability

**Comment by kapsakcj** [comment]
Thanks for the super quick turnaround on the review!

**Pull request merged**